openg2p/models/openg2p_beneficiary_api.py

```diff
@@ -0,0 +1,265 @@
+from odoo import fields
```

Comment:
Weren't all these Program and Beneficiary APIs already created in earlier PRs?

Reply:
They have been.
They are still visible in master.
And still these are shown as changes.